client/my-sites/media-library/external-media-header.jsx

```diff
@@ -123,11 +127,134 @@ class MediaLibraryExternalHeader extends React.Component {
 		return <span className="media-library__pexels-attribution">{ attribution }</span>;
 	}
 
+	renderCategoryFilter() {
+		const { translate } = this.props;
+		const categories = [
+			{
+				value: '',
+				label: translate( 'All categories' ),
+			},
+			{
+				value: 'animals',
+				label: translate( 'Animals' ),
+			},
+			{
+				value: 'arts',
+				label: translate( 'Arts' ),
+			},
+			{
+				value: 'birthdays',
+				label: translate( 'Birthdays' ),
+			},
+			{
+				value: 'cityscapes',
+				label: translate( 'Cityscapes' ),
+			},
+			{
+				value: 'crafts',
+				label: translate( 'Crafts' ),
+			},
+			{
+				value: 'food',
+				label: translate( 'Food' ),
+			},
+			{
+				value: 'fashion',
+				label: translate( 'Fashion' ),
+			},
+			{
+				value: 'food',
+				label: translate( 'Food' ),
+			},
+			{
+				value: 'flowers',
+				label: translate( 'Flowers' ),
+			},
+			{
+				value: 'gardens',
+				label: translate( 'Gardens' ),
+			},
+			{
+				value: 'holidays',
+				label: translate( 'Holidays' ),
+			},
+			{
+				value: 'houses',
+				label: translate( 'Houses' ),
+			},
+			{
+				value: 'landmarks',
+				label: translate( 'Landmarks' ),
+			},
+			{
+				value: 'landscapes',
+				label: translate( 'Landscapes' ),
+			},
+			{
+				value: 'night',
+				label: translate( 'Night' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Night Club' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).